pages/publishing/publishing/adobe-experience-platform.mdx

```diff
@@ -0,0 +1,81 @@
+import {Callout} from 'nextra/components'
+
+# Adobe Experience Platform
+
+*Publishing your tracking plan to the Adobe Experience Platform*
+
+![Adobe Logo](/images/publishing_adobe.svg)
+
+## Introduction
+
+The integration to [Adobe Experience Platform XDM system](https://experienceleague.adobe.com/en/docs/experience-platform/xdm/home) allows you to publish your Avo Tracking Plan as XDM Schemas to Adobe Experience Platform in form of XDM schemas. You can set up the integration to automatically publish on every Avo branch merge or manually trigger a publish. 
+
+## Configuring the Adobe Experience Platform integration
+
+To enable the direct integration with AEP, navigate to the Publishing Screen in your Avo workspace, click “Add Integration” and select “Adobe Experience Platform. 
+
+You will need to provide the following parameters from Adobe:
+
+- **Adobe Client Id**
+- **Adobe Organization ID**
+- **Adobe Client Secret**
+
+ These parameters can be found from the Adobe Developer Console -> Projects -> Workspaces -> Credentials. The URL should look like this: `https://developer.adobe.com/console/projects/{customerId}/{projectId}/workspaces/{workspaceId}/credentials/{credentialId}/details/{credentialsName}`.
+
+
+## Publishing to the Adobe Experience Platform
+
+To publish your Avo tracking plan to the Adobe Experience Platform, navigate to the AEP integration view in Avo and click the "Publish" button. Enable Auto Publishing to trigger an automatic publish whenever a branch has been merged.
+
+You can also download an XDM file, which you can import into AEP. 
+
+### What Is Included In The Publish to Adobe Experience Platform?
+
+On publish, Avo will fetch your existing XDM schemas, compare them with your Tracking Plan and publish new and updated XDM schemas. By default Avo will publish all events on the branch that you are on, but you can configure the publish to be [filtered by Sources, Destinations and Tags](/publishing/publishing/overview#filtering-events-for-publishing).
+
+> All entities published by Avo have `/avo/` in their ids, for example `https://ns.adobe.com/TENANT_ID/mixins/avo/my_properties`
+
+#### Events map to XDM Schemas
+
+The `Log Event` actions on Avo Events map to Adobe Experience Event schemas. 
```

Comment:
"The `Log Event` actions on Avo Events map to Adobe Experience Event schemas." -> "The `Log Event` action on Avo events map to Adobe Experience Event schemas."

Can we elaborate on what that means? An event in Avo that has a log event action is represented as a single AEP event schema?